sonar-plugin-api/src/main/java/org/sonar/api/batch/debt/DebtRemediationFunction.java

```diff
@@ -25,6 +25,7 @@
 
 /**
  * @since 4.3
+ * @deprecated since 6.5 debt model will soon be unavailable on batch side
  */
 public class DebtRemediationFunction {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)